specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -725,22 +728,27 @@
       "type": "object",
       "properties": {
         "id": {
+          "description": "The session id.",
```

Comment:
>session [](start = 30, length = 7)

The spark job's livy id.

Reply:
I think livy is necessary to make it clear.
Do you think "The livy id of the spark batch job." will be better?

---
In reply to: [455536463](https://github.com/Azure/azure-rest-api-specs/pull/10121#discussion_r455536463) [](ancestors = 455536463)